src/js/edu-benefits/schema/edu-benefits.js

```diff
@@ -0,0 +1,397 @@
+// Relocate options-for-select.
+const options = require('../../hca/utils/options-for-select');
+const _ = require('lodash');
+
+// const states = _.uniq(_.flatten(_.values(options.states)).map(object => object.value));
```

Comment:
It looks like they changed the way they're flattening the states list, so I don't think so.